packages/core/src/jsx.js

```diff
@@ -112,6 +128,23 @@ export const jsx: typeof React.createElement = function(
     }
   }
   newProps[typePropName] = type
+  if (process.env.NODE_ENV !== 'production') {
+    try {
+      throw new Error()
+    } catch (e) {
+      if (e.stack) {
+        // chrome
```

Comment:
🙈 they really should standardize this 😭 although there are horrible compat issues like this https://bugzilla.mozilla.org/show_bug.cgi?id=1498257#c11